builderberg-plugin/src/main/java/com/github/davidburkett/builderberg/generators/BuilderClassGenerator.java

```diff
@@ -116,6 +117,14 @@ private void generateWithSetters(final PsiClass builderClass, final List<PsiFiel
         }
     }
 
+    private PsiType getSanitizedType(final PsiType psiType) {
+        if (psiType instanceof PsiArrayType) {
+            return new PsiEllipsisType(psiType.getDeepComponentType());
```

Comment:
WTH is a deep component type? The JetBrains API sucks.

Reply:
https://dploeger.github.io/intellij-api-doc/com/intellij/psi/PsiType.html#getDeepComponentType-- Sounds like that should work.